scripts/create-image-tar.sh

```diff
@@ -28,7 +28,7 @@ done
 
 # Get operator-image
 operatorYamlPath="${parentDir}/../astraconnector_operator.yaml"
-pattern=' +image: netapp\/astra-connector-operator:([^\n ]*)'
```

Comment:
Not sure why this wasn't working, it was cutting off the end of my version when it was in the BUILD-BRANCH form